app/models/imports/podcast_rss_import.rb

```diff
@@ -132,22 +144,22 @@ def create_or_update_episode_imports!
     end
   end
 
-  def feed_description(feed)
-    result = [feed.itunes_summary, feed.description].find { |d| !d.blank? }
+  def feed_description
+    result = [channel[:itunes_summary], channel[:description]].find { |d| !d.blank? }
```

Comment:
From here on down, I'm just:

1. Changing accessors with hash `:[]` access instead of `Feedjira::Parser::Podcast` methods.
2. Just referencing the `channel` and `first_entry` stores directly rather than passing around this kind of ambiguous `feed` variable.